frontend/homework/1/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML Tags</title>
+    <link rel="stylesheet" href="styles">
+</head>
+<body>
+    <header>
+        <nav>
+            <h2>Kickdrum</h2>
+        </nav>
+    </header>
+    <main>
+    <div id="main">
+            <section>
+            <img src="https://images.squarespace-cdn.com/content/v1/587fc3e546c3c40b37363b71/1572274776794-X28FOBF0QUT8T1UPBGHM/Home-Header-Opt-1-Dark.png?format=2500w" style="height: 100%;width: 100%;" alt="bacakground image for first page">
+           
+                <h4>SERVICES</h4>
+                <h4>RESULTS</h4>
+                <h4>TEAM</h4>
+                
+                <h2>KICKDRUM</h2>
+                
+                    <h4>RESOURCES</h4>
+                <h4>CAREERS</h4>
+                <h4>CONTACT</h4>
+            <div id="center">
+                <h2>Mitigate technical risk.</h2>
+                <h1>Maximize business impact.</h1>
+                <h3>We partner with technology investors and executives
+                    to accelerate enterprise value.</h3>
+            </div>
+        </section>
+        <section>
+            <!-- table  -->
```

Comment:
Remove unnecessary comments
If you have added it for the sake of requirement then I would suggest adding comments to explain the code section